certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -206,6 +206,149 @@
         },
         "x-ms-visibility": "internal"
       }
+    },
+    "/api/v1/webhooks/azure-logic-apps": {
+      "get": {
+        "description": "CyberProof Triggers List.",
+        "summary": "CyberProof Triggers List",
+        "operationId": "CPGetTriggers",
+        "responses": {
+          "200": {
+            "description": "List of triggers",
+            "schema": {}
```

Comment:
@DavidMeu Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

Reply:
@sriyen-msft But this is internal operation